thingsboard_gateway/connectors/ble/device.py

```diff
@@ -105,11 +105,24 @@ async def timer(self):
         self.last_polled_time = time()
 
         while True:
-            if time() - self.last_polled_time >= self.poll_period:
-                self.last_polled_time = time()
-                await self.__process_self()
-            else:
-                await asyncio.sleep(.2)
+            try:
+                if time() - self.last_polled_time >= self.poll_period:
+                    self.last_polled_time = time()
+                    await self.__process_self()
+                else:
+                    await asyncio.sleep(.2)
+            except Exception as e:
+                log.exception(e)
+
+                try:
+                    await self.client.disconnect()
+                except Exception as err:
+                    log.exception(err)
+
+                while not self.stopped and not self.client.is_connected:
+                    await self.connect_to_device()
+
+                    sleep(.2)
```

Comment:
What if we cannot connect to device again? How  many tries the gateway will do and how often?